docs/_docs/user-interface/evolution.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Evolution
+permalink: /docs/evolution/
+---
+
+### Compute evolution
+
+You can use this step if you need to compute the row-by-row evolution of a value
+column, based on a date column. It will output 2 columns: one for the evolution
+in absolute value, the other for the evolution in percentage.
+
+You must be careful that the computation is scoped so that there are no dates
+duplicates (so that any date finds no more than one previous date). That means
+that you may need to specify index columns to make any date unique by index. You
+should use the `uniqueIndexCols` parameter to specifiy what series of columns
+allows to build a unique index (by concatenation with the date column).
+If index is not unique and more than one previous date is found, you will get
+an error at row level ("Error: More than one previous date found for the
+specified index columns"). Please refer to example 2 and 3 below for a concrete
+illustration.
+
+**This step is supported by the following backends:**
+
+- Mongo 4.0
+- Mongo 3.6
+
+#### Where to find this step?
+
+- Widget `Compute`
+- Search bar
+
+#### Options reference
+
+<img src="../../img/docs/user-interface/evolution_step_form.jpg" width="350" />
+
+- `Date column`: the date column (must be of type date) that will bbe used as a
```

Comment:
```suggestion
- `Date column`: the date column (must be of type date) that will be used as a
```

Reply:
Why don't we automatically parse the date in this step ? It will save the step to systematically "parse date" before.